### ISSUE TITLE
Can't get this (or any other custom render feature) working

### ISSUE THREAD

**Issue by klattimer** [yahiaetman/URPCustomPostProcessingStack#2]
**Describe the bug**
Not sure what's going on here, but this seems to be the most complete example in this space, my AddRenderPasses method is never called in any ScriptableRenderFeature I add, I've added them to the URP pipeline as render features, I've set up the volumes etc... 

No idea why unity isn't actually calling into the effect but as far as I can see it's in the ScriptableRenderFeature subclass. I like the way this project is set up so I'd love to get it working and add my own shaders to it. 

Any help much appreciated.

**Comment by AstricleGames5411** [comment]
Hey just change all afterpostprocess to BeforePostProcess hope this helps!

**Comment by klattimer** [comment]
Yeah, you're right... but I actually want to have my shaders overlay above the depth of field... Otherwise the overlay just gets blurred too...
